pallets/nfts/src/features/approvals.rs

```diff
@@ -54,7 +54,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			Self::is_pallet_feature_enabled(PalletFeature::Approvals),
 			Error::<T, I>::MethodDisabled
 		);
-		let mut details = Item::<T, I>::get(collection, item).ok_or(Error::<T, I>::UnknownItem)?;
+		let mut details =
+			Item::<T, I>::get(collection.clone(), item).ok_or(Error::<T, I>::UnknownItem)?;
```

Comment:
Can those not by a reference?

Reply:
I believe yes. I mainly make simple changes for the purpose of making code successfully compiled.